daemon/exec_windows.go

```diff
@@ -0,0 +1,8 @@
+// +build windows
+
+package daemon
+
+// checkExecSupport returns an error if the exec driver does not support exec
+func checkExecSupport(DriverName string) error {
+	return nil
```

Comment:
@jhowardmsft we could add a note to the godoc and finish the sentence like:

    ... or nil if it is supported.